picard/ui/options/metadata.py

```diff
@@ -92,6 +92,8 @@ class MetadataOptionsPage(OptionsPage):
         ListOption('setting', 'script_exceptions', [], title=N_("Translation script exceptions")),
         BoolOption('setting', 'release_ars', True, title=N_("Use release relationships")),
         BoolOption('setting', 'track_ars', False, title=N_("Use track and release relationships")),
+        BoolOption('setting', 'disable_date_sanitize', False, title=N_("Disable date sanitization for APE and Vorbis tags")),
+        ListOption('setting', 'formats_to_disable_date_sanitize', [], title=N_("Formats to disable date sanitize")),
```

Comment:
Since each format should appear only once I guess a set() is a better type for this option.